src/libs/actions/SignInRedirect.ts

```diff
@@ -29,6 +30,7 @@ function clearStorageAndRedirect(errorMessage?: string): Promise<void> {
     }
 
     return Onyx.clear(keysToPreserve).then(() => {
+        Policy.clearAllPolicies();
```

Comment:
I think the placement for the method is fine here, but I have never touched this part of the code, so I am not sure if we have any preference here.

Will defer to @yuwenmemon  for this.

Reply:
Seems fine to me